features/step_definitions/common/security-steps.js

```diff
@@ -111,3 +112,22 @@ Given('The user {string} should match:', async function (userId, dataTable) {
 
   should(user.content).match(expectedContent);
 })
+
+Given('I create an {string} file with content:', async function (filename, dataTable) {
```

Comment:
The step name should be more specific since it's specific to `userMappings` dump files